operator/config/rbac/role.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   creationTimestamp: null
-  name: keda-http-addon-manager-role
+  name: keda-http-manager-role
```

Comment:
@ajanth97 did the `mage generate` command change this?

Reply:
I used mage dockerBuildAll, I am not really sure what caused this change :(